flask_backend_design.py

```diff
@@ -0,0 +1,272 @@
+        if balance[1] >= float(amount):
+            cursor.execute("INSERT INTO Withdrawl_bankdemo \
+                            (AMount,WithdrawlDate,accountnumber,\
+                             Createdat,Createdby,Modifiedat,Modifiedby) \
+                             VALUES (?,?,?,?,?,?,?)",
+                           amount, date, accountnumber,
+                           date, CurrentUser, date, CurrentUser)
+            cursor.commit()
+            return redirect(url_for("account_balance",
+                                    accountnumber=accountnumber,
+                                    amount=amount,
+                                    transactiontype=transactiontype))
+        else:
+            return "Your available balance is {} .\
+            You can't withdraw more than this amount".format(balance[1])
+
+
+""" Performs Deposit process """
+
+
+@app.route('/deposit/<accountnumber>/<amount>/<transactiontype>')
+def deposit_amount(accountnumber, amount, transactiontype):
+    cursor.execute("INSERT INTO Deposit_bankdemo \
+    (AMount,DepositDate,accountnumber,Createdat,\
+    Createdby,Modifiedat,Modifiedby) \
+VALUES (?,?,?,?,?,?,?)", amount, date, accountnumber,
+                   date, CurrentUser, date, CurrentUser)
+    cursor.commit()
+    return redirect(url_for("account_balance",
+                            accountnumber=accountnumber,
+                            amount=amount,
+                            transactiontype=transactiontype))
+
+
+""" Update the balance based on Transaction type \
+    and displays the details for the same"""
+
+
+@app.route('/balance/<accountnumber>/<amount>/<transactiontype>')
+def account_balance(accountnumber, amount, transactiontype):
+    if transactiontype == 'deposit':
+        cursor.execute("INSERT INTO Transactions_bankdemo \
+        (accountnumber,Transactiondate,AMount,Transactiontype,Createdat,\
+        Createdby,Modifiedat,Modifiedby) \
+VALUES (?,?,?,?,?,?,?,?)", accountnumber, date, amount,
+                       transactiontype, date, CurrentUser, date, CurrentUser)
+        cursor.commit()
+        cursor.execute("Update AccountBalance_bankdemo SET Balance=balance+?,\
+                        Modifiedat=?,Modifiedby=? Where accountnumber=?",
+                       amount, date, CurrentUser, accountnumber)
+        cursor.commit()
+        cursor.execute("SELECT D.accountnumber,convert(float,d.Amount)Amount,\
+        CONVERT(DATETIME,d.DepositDate)DepositDate,\
+        convert(float,B.Balance)Balance \
+        FROM Deposit_bankdemo d WITH(Nolock)\
+        INNER JOIN AccountBalance_bankdemo B WITH(Nolock)\
+            ON d.accountnumber=B.accountnumber\
+        Where d.accountnumber=? ", accountnumber)
+        rows = cursor.fetchall()
+        if rows:
+            for row in rows:
+                return jsonify({"accountnumber": row[0],
+                                "DepositedAmount": row[1],
+                                "DepositDate": row[2],
```

Comment:
Remaining comments same as above. use json.dumps, abstract the queries, returning after first iteration etc.